ospf6d/ospf6_lsdb.h

```diff
@@ -24,6 +24,7 @@
 #include "prefix.h"
 #include "table.h"
 #include "ospf6_route.h"
+#include "lib/json.h"
```

Comment:
I have added this header here, because the function declarations below use the json_object data structure.

Reply:
it can be removed.

Reply:
Resolved